src/braket/circuits/quantum_operator.py

```diff
@@ -103,33 +102,45 @@ def ascii_symbols(self) -> tuple[str, ...]:
 
     @property
     def name(self) -> str:
-        """
-        Returns the name of the quantum operator
+        """Returns the name of the quantum operator
 
         Returns:
             str: The name of the quantum operator as a string
         """
         return self.__class__.__name__
 
-    def to_ir(self, *args, **kwargs) -> Any:
+    def to_ir(self, *args: Any, **kwargs: Any) -> Any:
         """Returns IR representation of quantum operator.
 
+        Args:
```

Comment:
It's weird to need to do this. Can we not configure ruff to ignore these checks for functions that are not implemented?

Reply:
I think it would be better if the `args` and `kwargs` are removed in this case. This is probably a good indicator that functions that aren't implemented should not have these.